packages/api/endpoints/collections.js

```diff
@@ -27,14 +28,16 @@ const log = new Logger({ sender: '@cumulus/api/collections' });
  * @returns {Promise<Object>} the promise of express response object
  */
 async function list(req, res) {
-  const { getMMT, ...queryStringParameters } = req.query;
-  const collection = new Collection(
+  const { getMMT, includeStats, ...queryStringParameters } = req.query;
+  const CollectionClass = includeStats === 'true' ? Collection : BaseCollection;
```

Comment:
Can we pass `includeStats` to `Collection` class, and let the class decide if the stats should be included or not?

Reply:
I could do that. I actually thought this was cleaner. But I'm fine changing it if you think that is a better way to do it.

Reply:
I prefer letting `Collection` class worry about the `includeStats` flag.

Reply:
Ok, I'm putting that up already.  Thanks.